### PR TITLE
feat(judicial-system): Who filed court document

### PR DESCRIPTION
**Pull request by oddsson**
# Who filed court document

https://app.asana.com/0/1199153462262248/1201614149125809/f

## What

Add the ability for court users to say who filed a court document, the defendant or prosecutor.

## Screenshots / Gifs

https://user-images.githubusercontent.com/3789875/158596856-29011a90-056c-4590-94e7-4695d56da8dd.mov
Screenshot 2022-03-16 at 12 50 27
Screenshot 2022-03-16 at 12 52 03

## Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

### CODE REVIEW

File: apps/judicial-system/backend/src/app/modules/case/models/case.model.ts [hunk 1]
```diff
@@ -532,7 +533,7 @@ export class Case extends Model<Case> {
     allowNull: true,
   })
   @ApiProperty()
-  courtDocuments?: string[]
+  courtDocuments?: CourtDocument[]
```
[the remark this answers is not in the document]
Comment (gudjong) [reply apps/judicial-system/backend/src/app/modules/case/models/case.model.ts:536]: Probably not, I will take a look

[the remark this answers is not in the document]
Comment (gudjong) [reply apps/judicial-system/backend/src/app/modules/case/models/case.model.ts:536]: Probably not, I will take a look

[the remark this answers is not in the document]
Comment (gudjong) [reply apps/judicial-system/backend/src/app/modules/case/models/case.model.ts:536]: Fixed

**Review by gudjong — APPROVED** [review]

**Review by gudjong — APPROVED** [review]

**Review by albinagu — APPROVED** [review]
lgtm

File: apps/judicial-system/web/src/utils/formatters.ts [hunk 2]
```diff
@@ -8,7 +8,9 @@ import { validate } from './validate'
 
 export const parseArray = (property: string, array: string[]) => {
   try {
+    // TODO: fix - need to escape/stringify ${a}
```
Comment (albinagu) [inline apps/judicial-system/web/src/utils/formatters.ts:11]: is it something that could this be fixed in this pr rather than marked as todo?

**Pull request merged**